### DISCUSSION TITLE
Is there a way to render any kind of card in chat?

### DISCUSSION THREAD

**Discussion by opensourcekam** [flyerhq/flutter_chat_ui#606] [Q&A] [answered]
Great work y'all I'm wondering if there is a way to render any type of card within the chat context? I.e. intercept the rendering engine and tell it to render X component as a preview instead of the underlying component.

**Comment by demchenkoalex** [comment, marked as answer]
hmm to mind comes only `customMessageBuilder` if you want a specific message to be custom, otherwise to add anything between messages, or in general, maybe better to fork and adjust to your use case
